tests_integration/runPrettier.js

```diff
@@ -75,14 +55,15 @@ function runPrettier(dir, args, options) {
     require("../bin/prettier");
     status = status || process.exitCode || 0;
   } catch (error) {
-    stderr += error.message;
     status = 1;
+    stderr += error.message;
   } finally {
-    process.exitCode = originalExitCode;
-    process.stdin.isTTY = originalIsTTY;
-    process.argv = originalArgv;
     process.chdir(originalCwd);
+    process.argv = originalArgv;
+    process.exitCode = originalExitCode;
+    process.stdin.isTTY = originalStdinIsTTY;
```

Comment:
Do these lines (and some of the ones further up in the file) need to be rearranged? Just trying to reduce unnecessary diff churn if possible.

Looks good otherwise!

Reply:
The order does not matter, but I'd like to make them in the same order since it'd be easy to check if they paired correctly:

```
a
b
c
d

b
d
c
a
```

vs

```
a
b
c
d

a
b
c
d
```